Gemfile

```diff
@@ -9,6 +9,7 @@ gem "rails", "~> 6.0.2"
 gem "asset_sync"
 gem "bootsnap", "~> 1"
 gem "dynamoid"
+gem "faraday"
```

Comment:
I guess we should add a version for this, I'm not sure why the asset-sync / dynamoid ones don't have versions set.

Reply:
Updated and fixed up.